runtime/codert_vm/decomp.cpp

```diff
@@ -1408,7 +1408,20 @@ jitDataBreakpointAdded(J9VMThread * currentThread)
 			/* Make all future compilations inline the field watch code */
 			jitConfig->inlineFieldWatches = TRUE;
 		} else {
+			/* jitClassesRedefined expects the caller to hold the class unload mutex exclusively. */
+#if defined(J9VM_JIT_CLASS_UNLOAD_RWMONITOR)
+			omrthread_rwmutex_enter_write(vm->classUnloadMutex);
+#else
```

Comment:
`#else` also needs the comment.

```suggestion
#else /* defined(J9VM_JIT_CLASS_UNLOAD_RWMONITOR) */


```